src/features/hotspots/transfers/TransferRequests.tsx

```diff
@@ -0,0 +1,70 @@
+import Balance, { CurrencyType, NetworkTokens } from '@helium/currency'
+import {
+  deleteWallet,
+  getWallet,
+  postWallet,
+} from '../../../utils/walletClient'
+
+type TransferHTTP = {
+  amount_to_seller: number // in bones
+  partial_transaction: string
+  gateway: string
+  seller: string
+  buyer: string
+}
+
+export type Transfer = {
+  amountToSeller?: Balance<NetworkTokens>
+  partialTransaction: string
+  gateway: string
+  seller: string
+  buyer: string
+}
+
+export const createTransfer = async (
```

Comment:
not a big deal, but just wondering if this is the right place for this file? would it be more at home in `utils` or even as a redux slice?

Reply:
Hmm yeah good point, I probably should move the data to redux as right now its integrated into the components. I think in regards to time I would leave it as it is for now. Most of the data is already in the SendComponent anyway. 

I kind of like having all the transfer stuff in one folder so its easier to find. I could move this to utils though?

Reply:
Going to merge it now so people can start testing and we can address this is a separate PR if we want to move it to redux